sashimi/hardware/lasers/__init__.py

```diff
@@ -0,0 +1,54 @@
+from enum import Enum
+
+
+class LaserState(Enum):
+    ON = 1
+    OFF = 2
+
+
+class LaserException(Exception):
+    pass
+
+
+class LaserWarning(Warning):
+    pass
+
+
+class AbstractLaser:
+    def __init__(self, port):
+        self.port = port
+        self._status = LaserState.OFF
```

Comment:
since it can (I guess) only be on or off, why not just have a bool? (self._laser_on)

Reply:
Other lasers could be ON, OFF, STAND-BY. It is more general to have a class that implements this. One could inherit this class and further introduce new states in other lasers. For cobolt it is true there are only 2 states